cms/templates/cms/wizards/base.html

```diff
@@ -15,6 +15,7 @@
 {% block extrahead %}
     {{ block.super }}
     <script src="{% static "cms/js/dist/bundle.admin.base.min.js" %}" type="text/javascript"></script>
+    {{ form.media }}
     <script>
     $(function () {
```

Comment:
we might want to move this also into ``cms.wizards.js`` and include it globally.